jsettlers.logic/src/jsettlers/logic/buildings/Building.java

```diff
@@ -81,12 +84,12 @@
 	private static final EPriority[] SUPPORTED_PRIORITIES_FOR_CONSTRUCTION = new EPriority[] { EPriority.LOW, EPriority.HIGH, EPriority.STOPPED };
 	private static final EPriority[] SUPPORTED_PRIORITIES_FOR_NON_WORKERS = new EPriority[0];
 
-	private static final ConcurrentLinkedQueue<Building> allBuildings = new ConcurrentLinkedQueue<Building>();
+	private static final ConcurrentLinkedQueue<Building> ALL_BUILDINGS = new ConcurrentLinkedQueue<Building>();
```

Comment:
Well, the JLS states exactly: 
```
The names of constants in interface types should be, and final variables of class types may conventionally be, a sequence of one or more words, acronyms, or abbreviations, all uppercase, with components separated by underscore "_" characters.
```

=> We can choose the naming of `static final OBJECT_TYPE`. Therefore I think your idea to only use uppercase for really constant values is a good idea. I will change the naming.

@homoroselaps: Thnaks for bringing this up.